mdtraj/core/topology.py

```diff
@@ -1295,13 +1295,15 @@ class Chain(object):
         Iterator over all Atoms in the Chain.
     """
 
-    def __init__(self, index, topology):
+    def __init__(self, index, topology, chain_id = None):
         """Construct a new Chain.  You should call add_chain() on the Topology instead of calling this directly."""
         # The index of the Chain within its Topology
         self.index = index
         # The Topology this Chain belongs to
         self.topology = topology
         self._residues = []
+        # PDB format chainID
+        self.chain_id = chain_id
```

Comment:
please add a description of this attribute to the docstring above

Reply:
done